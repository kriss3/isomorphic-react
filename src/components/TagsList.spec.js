import React from 'react';
import TagsList from './TagsList';
import renderer from 'react-test-renderer';


describe('The Tags List', ()=>{
    it('renders as expected', ()=>{
        const tree = renderer
        .create(<TagsList tags={[`css`,`html`,`swift`]}/>)
        .toJSON();

        expect(tree).toMatchSnapshot();
    });
});