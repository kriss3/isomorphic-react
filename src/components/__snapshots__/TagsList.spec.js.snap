// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Tags List renders as expected 1`] = `
<div>
  <code>
    css
  </code>
  <code>
    html
  </code>
  <code>
    swift
  </code>
</div>
`;
